### PR TITLE
[Doc] Fix the docstring of builtin functions.

### PR DESCRIPTION
**Pull request by zheng-da**
## Description
Provide better explanation of builtin functions.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

File: python/dgl/function/message.py [hunk 1]
```diff
@@ -168,15 +168,17 @@ def copy_e(e, out):
 
 def _gen_message_builtin(lhs, rhs, binary_op):
     name = "{}_{}_{}".format(lhs, binary_op, rhs)
-    docstring = """Builtin message function that computes message by performing
-    binary operation {} between {} feature and {} feature.
+    docstring = """Builtin message function that computes a message on an edge
+    by performing element-wise {} between features of {} and {}
+    if the features have the same shape; otherwise, it first broadcasts the features
```
Comment (yzh119) [inline python/dgl/function/message.py]: I think we elaborate more on broadcasting behaviour: https://github.com/dmlc/dgl/blob/master/python/dgl/kernel.py#L75-L85

Comment (zheng-da) [reply python/dgl/function/message.py]: any reference to numpy broadcasting semantics?

File: CMakeLists.txt [hunk 2]
```diff
@@ -23,7 +23,7 @@ endif()
 # and add set(OPTION VALUE) to override these build options.
 # Alernatively, use cmake -DOPTION=VALUE through command-line.
 dgl_option(USE_CUDA "Build with CUDA" OFF)
-dgl_option(USE_OPENMP "Build with OpenMP" ON)
+dgl_option(USE_OPENMP "Build with OpenMP" OFF)
```
Comment (yzh119) [inline CMakeLists.txt]: Should we change this? I suppose we just need to customize this in `config.cmake`

File: python/dgl/function/message.py (hunk 1 above)
Comment (yzh119) [reply python/dgl/function/message.py]: https://docs.scipy.org/doc/numpy/user/basics.broadcasting.html#general-broadcasting-rules

**Pull request merged**